sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -217,6 +217,12 @@ def test_sin_rewrite():
     assert sin(x).rewrite(cos) == cos(x - pi / 2, evaluate=False)
     assert sin(x).rewrite(sec) == 1 / sec(x - pi / 2, evaluate=False)
     assert sin(cos(x)).rewrite(Pow) == sin(cos(x))
+    assert sin(x).rewrite(besselj).subs(x,pi/4).n() == sin(x).subs(x,pi/4).n()
```

Comment:
Can you use standard PEP 8 whitespace rules here and throughout the diff? This means one space after a comma, like

```suggestion
    assert sin(x).rewrite(besselj).subs(x, pi/4).n() == sin(x).subs(x, pi/4).n()
```